core/src/main/java/com/netflix/msl/msg/MessageBuilder.java

```diff
@@ -84,7 +84,14 @@ public static long decrementMessageId(final long messageId) {
         return (messageId == 0) ? MslConstants.MAX_LONG_VALUE : messageId - 1;
     }
 
-    protected MessageBuilder() {}
+    /**
+     * <p>Create a new message builder.</p>
```

Comment:
Add Javadoc here indicating that subclasses _must_ call this protected constructor and that they _should_ call `initializeMessageBuilder()`.